### ISSUE TITLE
data_readers.clj default

### ISSUE THREAD

**Issue by vemv** [webnf/lein-collisions#4]
As the README notes, data_readers.clj is a common case for a resource name that can be repeated.

Checking out https://clojure.org/reference/reader#tagged_literals , that appears to be expected.

Would you consider adding it to the default set of excluded resource names?

Thanks - V

**Comment by vemv** [comment]
Same for `COPYRIGHT` e.g. 

```
COPYRIGHT -- 2  collisions:
  -- /Users/vemv/.m2/repository/com/cognitect/http-client/0.1.106/http-client-0.1.106.jar
  -- /Users/vemv/.m2/repository/com/cognitect/anomalies/0.1.12/anomalies-0.1.12.jar
```

seems close enough in spirit to current default files

**Comment by bendlas** [comment]
Yep, I think I created this plugin before learning about the merging behavior of tagged literals. Thanks for bringing it up! Including `COPYRIGHT` into default exclusions also seems like a good idea.
